build/configs/rtl8720e/scripts/rlx8720e_img2_is.ld

```diff
@@ -67,9 +67,9 @@ MEMORY
 	KM4_MSP_RAM_S (rw) :                    ORIGIN = 0x30008000, LENGTH = 0x3000A000 - 0x30008000	/* KM4 MSP_S RAM: 8k */
 	KM4_BOOTLOADER_RAM_S (rwx) :            ORIGIN = 0x3000A020, LENGTH = 0x30012000 - 0x3000A020	/* KM4 BOOT Loader RAM: 32k */
 	KM4_FLOADER_RAM_S (rwx) :               ORIGIN = 0x3000A020, LENGTH = 0x30020000 - 0x3000A020	/* KM4 ImgTool Flash Loader RAM: 88k */
-	KM4_BD_RAM_NSC (rwx) :                  ORIGIN = 0x60347020, LENGTH = 0x60348000 - 0x60347020	/* KM4 BD RAM NSC: 4k */
-	KM4_BD_RAM_ENTRY (rwx) :                ORIGIN = 0x60348000, LENGTH = 0x6034C000 - 0x60348000	/* KM4 BD RAM ENTRY: 16k */
-	KM4_BD_RAM_S (rwx) :                    ORIGIN = 0x7034C020, LENGTH = 0x7039C000 - 0x7034C020	/* KM4 BD RAM S: 320k */
+	KM4_BD_RAM_NSC (rwx) :                  ORIGIN = 0x60351020, LENGTH = 0x60352000 - 0x60351020	/* KM4 BD RAM NSC: 4k */

```

Comment:
can you please change commit message to build/configs/rtl8720e for clearly knowing which board this PR for